suites/quincy/nvmeof/tier-1_nvmeof_functl_Regression.yaml

```diff
@@ -123,7 +126,46 @@ tests:
       name: Test to perform Data Integrity test over NVMe-OF targets
       polarion-id:
 
-  # Cleanup Test Case
+# Run IOs using multiple-initiators against multi-subsystems based NVMe-OF targets using fio tool - Multi gateways in the cluster
+  - test:
+      abort-on-fail: true
+      config:
+        gw_node: node8                        # This covers multiple NVMEoF Gateway nodes in the cluster
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        install: true                           # Run SPDK with all pre-requisites
+        subsystems:                             # Configure subsystems with all sub-entities
+          - nqn: nqn.2016-06.io.spdk:cnode3
+            serial: 3
+            bdevs:
+              count: 1
+              size: 10G
+            listener_port: 5003
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode4
+            serial: 4
+            bdevs:
+              count: 1
+              size: 10G
+            listener_port: 5004
+            allow_host: "*"
+        initiators:                             # Configure Initiators with all pre-req
+          - subnqn: nqn.2016-06.io.spdk:cnode3
+            listener_port: 5003
+            node: node9
+          - subnqn: nqn.2016-06.io.spdk:cnode4
+            listener_port: 5004
+            node: node10
+      desc: Perform IOs on multi-subsystems with Multiple-Initiators
+      destroy-cluster: false
+      module: test_ceph_nvmeof_gateway.py
+      name: Test to run IOs using multiple-initiators against multi-subsystems
+      polarion-id:
+
+#  # Cleanup Test Case
```

Comment:
IMO, if we are targeting for whole setup cleanup, we would need to also clean up second gateway on node8 and its subsystems and initiators

Reply:
Its depending on us what to clean up, we can include whatever we want to clean up in the test case.  Its user depending